source/administration/concepts.rst

```diff
@@ -0,0 +1,162 @@
+
+The process of authenticating a user and verifying access to objects is known as the `TLS Handshake`.
+Once authenticated, TLS provides the cipher to encypt and then decrypt the transfer of information from the server to the requesting client.
+
+MinIO supports several methods of :ref:`Server-Side Encryption <minio-encryption-overview>`.
+
+Can I organize objects in a folder structure within buckets?
+------------------------------------------------------------
+
+MinIO is a flat object store, meaning there is no structure to a deployment other than the bucket(s).
+However, MinIO does allow for using a prefixing scheme with a ``/`` delimitor that can mimic a folder structure from traditional file systems.
+
+Prefixing involves prepending the name of an object with a fixed string.
+When you use the same prefix string on multiple objects, MinIO identifies those as similar or grouped objects.
+
+Consider an object in an ``images`` bucket named ``vacation/2019/october/17/mountain-hike.jpg``.
+MinIO treats this as an object ``mountain-hike.jpg`` in a nested series of locations:
+
+.. code-block::
+
+   vacation/
+      2019/
+         october/
+            17
+
+If you had a second object in the bucket named ``vacation/2019/october/17/waterfall.jpg``, you could find both objects by calling on the ``vacation/2019/october/17/`` prefix.
+
+Remember:
+
+- MinIO buckets do not actually arrange contents in folders.
+- You must always refer to a specifc object with its full name, including the prefix.
+- For MinIO objects, the prefix "path" is part of the object name.
+
+What tools does MinIO provide to manage where objects store?
+------------------------------------------------------------
+
+MinIO provides two features to manage where an object stores.
+These tools include replication and tiering.
+
+MinIO provides two types of replication to copy an object from one location to another.
+You can configure replication at either the :ref:`bucket level <minio-bucket-replication>` or at the :ref:`site level <minio-site-replication-overview>`.
+
+
+- Bucket level replication can function as either one-way, active-passive replication (such as for archival purposes) or as two-way, active-active replication to keep two buckets in sync with each other.
+- Site level replication functions as two-way, active-active replication to keep multiple data locations (such as different geographic data centers) in sync with one another.
+
+:ref:`Tiering rules <minio-lifecycle-management-tiering>` allow frequently accessed objects to store on hot or warm storage, which is typically more expensive but provides better performance.
+Less frequently accessed objects can move to cold storage.
+Cold storage often exchanges slower performance for a cheaper price.
+
+
+What tools does MinIO provide to manage object lifecycle?
```

Comment:
If we are keeping this, then the previous section can maybe specifically target "How do I use different storage types with MinIO?" because the answer *there* is to use tiering with hot/warm/cold MinIO deployments based on storage type.

Reply:
Oh - and this section should be ILM only.

Locking/Versioning is more "How does MinIO protect objects from accidental overwrite or deletion?"

Reply:
Split as suggested.